index.html

```diff
@@ -0,0 +1,206 @@
+        </figure>
+        <section class="beatles-p">
+          <p>Abbey Road is a thoroughfare located in the borough of Camden and the City of Westminster in London, running roughly northwest to southeast through St. John's Wood, near Lord's Cricket Ground. It is part of the B507. This road is best known for the Abbey Road Studios and the 1969 album, Abbey Road, by The Beatles.</p>
+          <p class="photocred">Photo: goo.gl/h44dc9</p>
+        </section>
+      </article>
+    </section>
+
+    <!-- border bottom line -->
+    <section class="immigration">
+      <figure>
+        <ul>
+          <li><img src="img/italian.png" alt=""></li> <!-- upsidedown heads -->
+          <li><img src="img/indian.png" alt=""></li>
+          <li><img src="img/asian.png" alt=""></li>
+          <li><img src="img/blonde.png" alt=""></li>
+        </ul>
+      </figure>
+      <article>
+        <header>
+          <h1>Border Bottom Line</h1>
+          <h3>Awful immigration laws from around the world.</h3>
+        </header>
+        <p>
+          <strong>Italy</strong> penalizes illegal immigrants with a find of 5,000-10,000 and allows immigration officials to detain them for up to 6 months.
+        <!-- bullet point --> &bull; <strong>Switzerland</strong> immediately deports all convicted criminals from other countries and potentially even their family members.
+        <!-- bulletpoint --> &bull; <strong>Australia</strong> detains all non-citizens without a valid visa.
+        <!-- bulletpoint --> &bull; <strong>Japan</strong> is allowed to pay $3,000 to each unemployed Latin Americn immigrant of Japanese descent and $2,000 to each of that unemployed worker's family members to return to their country of origin.
+        <!-- bulletpoint -->&bull; The <strong>United Arab Emirates</strong> prohibits foreigners from engaging in any sort of labor union-like activity, resulting in harsh living conditions, 80-hour work weeks, back-breaking manual labor, and below-minimum-wage pay.
+        </p>
+      </article>
+    </section>
+
+    <!-- transformers -->
+    <section class="transformers">
+
+      <figure>
+        <img src="img/transformers.png">
+      </figure>
+
+      <article>
+        <header>
+          <h1>Transformers</h1>
+          <h3>Revenge of the Fallen</h3>
+        </header>
+        <section class="transformers-p">
+          <p><em>A noisy, underplotted, and overlong special effects extravaganza that lacks a human touch.</em></p>
+        </section>
+
+        <!-- rating -->
+        <section class="rating">
+          <h3 class="gray">Rotten Tomato</h3>
+          <h1 class="gray">Rating</h1>
+          <h1 class="progress">20%</h1>
+          <progress min="0" max="100" value="20">20%</progress>
+        </section>
+
+        <!-- date -->
+        <section class="date">
+          <h5 class="gray premiere">World Premiere</h5>
+          <h3>Den Haag Theatre</h3>
+          <h3 class="gray">The Netherlands</h3>
```

Comment:
Cannot have the same header types in a section, also I really avoid classes like `gray` as much as possible. 

Why is this text gray? What is the semantic pattern you are applying presentation to?